services/web/server/src/simcore_service_webserver/users_api.py

```diff
@@ -112,17 +113,21 @@ async def update_user_profile(
 
 async def is_user_guest(app: web.Application, user_id: int) -> bool:
     """Returns True if the user exists and is a GUEST"""
-    db = get_storage(app)
-    user = await db.get_user({"id": user_id})
-    if not user:
-        logger.warning("Could not find user with id '%s'", user_id)
-        return False
+    engine: Engine = app[APP_DB_ENGINE_KEY]
```

Comment:
if I get it correctly you removed asyncpg from the webserver?

Reply:
only here. asyncpg shall only be used in the login plugin. At this point there are still pieces of it around ... slowly refactoring those.